### PR TITLE
[1237] Per provider enable/disable Apply sync

### PR DESCRIPTION
**Pull request by JonF1**
### Context
https://trello.com/c/Yo3PV3HD/1237-per-provider-enable-disable-apply-sync
### Changes proposed in this pull request
Added apply_sync_enabled field to providers table
Checkbox on Provider new page that sets value to true when selected
Add to feature test
Validation out of scope?
### Guidance to review
At /system-admin/providers/new
Screenshot 2021-03-11 at 21 03 26

### CODE REVIEW

**Comment by JonF1** [comment]
[[S] Per provider enable/disable Apply sync](https://trello.com/c/Yo3PV3HD/1237-s-per-provider-enable-disable-apply-sync)

File: app/views/system_admin/providers/new.html.erb [hunk 1]
```diff
@@ -12,6 +12,7 @@
 
       <%= f.govuk_text_field :name, label: { text: "Provider name", size: "s" }, width: 20, autocomplete: :disabled %>
       <%= f.govuk_text_field :dttp_id, label: { text: "DTTP ID", size: "s" }, width: 20, autocomplete: :disabled %>
+      <%= f.govuk_check_box :apply_sync_enabled, true, multiple: false, label: { text: "Import application data from Apply?<br><br>".html_safe, size: "s" } %>
```
Comment (jhenderson) [inline app/views/system_admin/providers/new.html.erb:15]: You can get rid of the line breaks and `html_safe` method call and just wrap the checkbox with `<div class="govuk-form-group"></div>.`

**Review by jhenderson — CHANGES_REQUESTED** [review]
Looks good, just a small change request.

File: app/views/system_admin/providers/new.html.erb (hunk 1 above)
Comment (JonF1) [reply app/views/system_admin/providers/new.html.erb]: Thanks John, I've fixed

**Review by jhenderson — APPROVED** [review]

**Pull request merged**